CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 1.1.10
+  - Fixed crash on shutdown if no DLQ messages were read
```

Comment:
```suggestion
  - Fix, avoid Logstash crash on shutdown if no DLQ files weren't created [#33](https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/34)
```

Reply:
believe it is a typo. should be /pull/33

Reply:
I've used the proposed wording in the new push, except for the type and double negative (which must have been mistakes)